src/main/webapp/features/shared/services/game/GameModel.service.js

```diff
@@ -0,0 +1,95 @@
+'use strict';
+
+angular.module('catan')
+        .factory('GameModel', ['$q', 'Remote', 'User', '$timeout', function ($q, Remote, User, $timeout) {
+
+            var GAME_PRIMARY_KEY = "gameId";
```

Comment:
as I understand, it was introduced to a local variable so that it can be fixed quickly when we change id on server?